config/webpack.config.prod.js

```diff
@@ -26,6 +26,7 @@ var nodeModulesPath = path.join(__dirname, '..', 'node_modules');
 var indexHtmlPath = path.resolve(__dirname, relativePath, 'index.html');
 var faviconPath = path.resolve(__dirname, relativePath, 'favicon.ico');
 var buildPath = path.join(__dirname, isInNodeModules ? '../../..' : '..', 'build');
+var publicPath = require('./package.json').homepage || '/';
```

Comment:
`package.json` might not be there. I think you need to use something like `path.resolve(__dirname, relativePath, 'package.json');`

Reply:
THANK U every much, try to config package.json with `"homepage":"./"` make it work on default app.